geoopt/manifolds/base.py

```diff
@@ -849,9 +849,28 @@ def _retr_transp(self, x, v, *more, u, t):
                 out = (y, self._transp2y(x, v, *more, y=y))
         return out
 
+    def rand_(self, x):
+        """
+        In-place implemetation for normal distribution
+        on manifold
+
+        Parameters
+        ----------
+        x : tensor
+            point on the manifold
+        Returns
+        -------
+        tensor
+            tensor of normally distributed points
+            on manifold
+        """
+        x.normal_()
```

Comment:
This is something what is not the good decision. It should at least be a part of random method as there is no point in doing normal in place beforehand. Moreover, I think that we should further jointly discuss the random api. My thought is that you should be able to pass some parameters to the method if it is applicable. The motivating case is initialization, large magnitudes are not always good.

Reply:
@newkozlukov @ferrine this one is still relevant